### DISCUSSION TITLE
In flows with a manual trigger could we also get the directus_users option?

### DISCUSSION THREAD

**Discussion by jimhaakman** [directus/directus#19087] [Feature Requests]
### Summary

I would like to trigger a manual flow inside a user.

### Motivation

For instance when you use Directus as a sort of CRM, or need to trigger a flow for customer support.

### Detailed Design

-

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by fitu86** [comment]
A la colección directus_users agrega un campo relacional a una colección con un flujo manual.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
